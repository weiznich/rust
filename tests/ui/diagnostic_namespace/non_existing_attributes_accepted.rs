// check-pass
#[diagnostic::non_existing_attribute]
//~^WARN Unknown diagnostic attribute
pub trait Bar {
}

#[diagnostic::non_existing_attribute(with_option = "foo")]
//~^WARN Unknown diagnostic attribute
struct Foo;

fn main() {
}
