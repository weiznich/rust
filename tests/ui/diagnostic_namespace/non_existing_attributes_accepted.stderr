warning: Unknown diagnostic attribute
  --> $DIR/non_existing_attributes_accepted.rs:3:15
   |
LL | #[diagnostic::non_existing_attribute]
   |               ^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unknown_diagnostic_attribute)]` on by default

warning: Unknown diagnostic attribute
  --> $DIR/non_existing_attributes_accepted.rs:8:15
   |
LL | #[diagnostic::non_existing_attribute(with_option = "foo")]
   |               ^^^^^^^^^^^^^^^^^^^^^^

warning: 2 warnings emitted

